event-processing/lambda/models/cleansed-event.ts

```diff
@@ -15,6 +15,21 @@ export interface ICleansedEvent {
     extensions?: unknown | undefined;
 }
 
+function CleanseEvidenceEvent(evidence: ICleansedEvidenceEvent) {
+
+    let fraudKeys = ['validityScore']
```

Comment:
I think we should define allowed (?) properties as a const at the top to decouple from this function